lib/engine/game/g_1880/minor.rb

```diff
@@ -8,6 +8,18 @@ module Game
     module G1880
       class Minor < Engine::Minor
         include ShareHolder
+        def num_shares_of(corporation, ceil: true)
+          num = percent_of(corporation).to_f / corporation.share_percent
+          ceil ? num.ceil : num
+        end
```

Comment:
Method is inherited from ShareHolder module. I don't think you need to redefine it here.